Source/KSCrash/Recording/Monitors/KSCrashMonitor_CPPException.cpp

```diff
@@ -184,13 +184,25 @@ catch(TYPE value)\
 #pragma mark - Public API -
 // ============================================================================
 
+static void initialize()
+{
+    static bool isInitialized = false;
+    if(!isInitialized)
+    {
+        isInitialized = true;
+        kssc_initCursor(&g_stackCursor, NULL, NULL);
```

Comment:
Perhaps a better place is inside `CPPExceptionTerminate()`, where we can look at `g_stackCursor`'s `advanceCursor` to see if we need an init? Don't know if there's any way we can mark the backtrace as explicitly missing, so that it looks better in the reports? Either way we should probably `KSLOG_WARN` about the situation. Are any of the KSStackCursor fields appropriate to detect that nothing was filled out? `currentDepth == 0`? `address == 0`?